src/ui/public/pager_control/components/tool_bar_pager_text/tool_bar_pager_text.html

```diff
@@ -1,3 +1,11 @@
 <div class="kuiToolBarText" data-test-subj="toolBarPagerText">
-  {{ toolBarPagerText.startItem | number }}&ndash;{{ toolBarPagerText.endItem | number }} of {{ toolBarPagerText.totalItems | number }}
+  <span
+    i18n-id="common.ui.pagerControl.pagesCountLabel"
+    i18n-default-message="{startItem}&ndash;{endItem} of {totalItems}"
+    i18n-values="{
+      startItem: (toolBarPagerText.startItem | number),
```

Comment:
I thought we cannot use filters inside values? Wasn't that the issue just in another PR, why you extracted the `hitsMatchingFilter` out into the controller?

Reply:
We can use filters inside values, only we need:
- wrap value and filter in parentheses, for example `startItem: (toolBarPagerText.startItem | number)`,
- filter can't  be with parameter.

Otherwise our tool can't parse this values.

And I didn't find `hitsMatchingFilter` in code, could you please give me a link to it.

Reply:
https://github.com/elastic/kibana/pull/28653/files#diff-bdb1a82c694830a9c68e3f682f8eaa08L25 this was extracted into the controller, because it was not working in HTML, but might be, that just an additional parenthesis around it would have helped. But then I am fine with that.